config/config.go

```diff
@@ -33,15 +39,32 @@ type IngressConfig struct {
 func Get() *IngressConfig {
 
 	options := viper.New()
-	options.SetDefault("MaxSize", 10*1024*1024)
-	options.SetDefault("Port", 3000)
-	options.SetDefault("StageBucket", "available")
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
+		cfg := clowder.LoadedConfig
+		topic, ok := clowder.KafkaTopics["platform.upload.advisor"]
+		if !ok {
+			log.Fatal("topic not found")
+		}
+		options.SetDefault("Port", cfg.WebPort)
+		options.SetDefault("KafkaBrokers", fmt.Sprintf("%s:%v", cfg.Kafka.Brokers[0].Hostname, *cfg.Kafka.Brokers[0].Port))
+		options.SetDefault("KafkaGroupID", cfg.Kafka.Topics[0].ConsumerGroup)
+		options.SetDefault("KafkaTrackerTopic", topic.Name)
+		options.SetDefault("MinioEndpoint", fmt.Sprintf("%s:%d", cfg.ObjectStore.Hostname, cfg.ObjectStore.Port))
+		options.SetDefault("MinioAccessKey", *cfg.ObjectStore.AccessKey)
+		options.SetDefault("MinioSecretKey", *cfg.ObjectStore.SecretKey)
+		options.SetDefault("StageBucket", cfg.ObjectStore.Buckets[0].Name)
```

Comment:
There's only ever one bucket requested by ingress, right?  If so, then doing this is okay.